addon-test-support/index.js

```diff
@@ -1,7 +1,12 @@
 import { getContext } from '@ember/test-helpers';
-import { _enableFeature } from './-private/enable-feature';
+import { _enableFeature, _disableFeature } from './-private/toggle-feature';
 
 export function enableFeature(featureName) {
   let { owner } = getContext();
   _enableFeature(owner, featureName);
 }
+
+export function disableFeature(featureName) {
```

Comment:
`enableFeature` has `_enableFeature` extracted only because `_enableFeature` is used by the older `withFeature` helper -- there's no reason to do the same with `disableFeature`. I suggest splitting the two helpers into separate files (rather than toggle-feature), and implementing the whole of `disableFeature` in a `disable-feature.js` than can be reexported here `export { disableFeature } from './-private/disable-feature';`